css/style.css

```diff
@@ -0,0 +1,230 @@
+*{
+    scroll-behavior: smooth;
+}
+#nav-bar{
+    position: fixed;
+    top: 0px;
+    right: 0px;
+    padding: 10px 0px;
+    max-width:40vw;
+    display: block;
+    
+    
+    
+    
+    
+    
+}
+
+
+
```

Comment:
Please kindly remove all white spaces in your CSS files. You have too many of them.